LocalSettings.php

```diff
@@ -244,7 +244,29 @@ function wfShouldEnableSwift( $dbname ) {
 	'wgArticlePlaceholderRepoApiUrl' => [
 		'default' => 'https://www.wikidata.org/w/api.php',
 	],
-
+	
+	// Babel
+	'wgBabelCategoryNames' => [
+		'default' => [
```

Comment:
This isn't the right default
https://github.com/wikimedia/mediawiki-extensions-Babel/blob/master/extension.json#L66

Reply:
> This isn't the right default https://github.com/wikimedia/mediawiki-extensions-Babel/blob/master/extension.json#L66

done